mypy/main.py

```diff
@@ -7,8 +7,8 @@
 import sys
 import time
 
-from typing import Any, Dict, IO, List, Optional, Sequence, Tuple, TextIO, Union
-from typing_extensions import Final, NoReturn
+from typing import Any, Dict, IO, List, NoReturn, Optional, Sequence, Tuple, TextIO, Union
+from typing_extensions import Final
```

Comment:
I think we might still support 3.6.1
```suggestion
from typing import Any, Dict, IO, List, Optional, Sequence, Tuple, TextIO, Union
from typing_extensions import Final, NoReturn
```